### ISSUE TITLE
Repeat elements in one dimension a nonuniform number of times?

### ISSUE THREAD

**Issue by alex2awesome** [arogozhnikov/einops#89]
Hey great framework! 

This might be either simple or impossible... I'm just wondering, is it possible to repeat elements along an axis in differing amounts?

i.e.:

```
input = [
    [a, b],
    [c, d]
]

desired_output=[
    [a, a, b, b, b],
    [c, c, d, d, d],
]
```

An inelegant solution is:

```
t = [
    repeat(input[:, 0], 'i -> i c', c=2),
    repeat(input[:, 1], 'i -> i c', c=3)
]
torch.cat(t, dim=1)
```

**Comment by arogozhnikov** [comment]
Hi @alex2awesome, 
currently there is no such functionality in einops.

The best alternative so far is what you wrote (inelegant solution)
